js/RotorPirates.js

```diff
@@ -0,0 +1,170 @@
+// Ported to JS by Robert Miroszewski based on awesome work by Chris Simpson ;)
+
+
+// defaults
+// RF
+rfRate = 400
+rfExpo = 50
+rfAcro = 140
+
+// BF
+bfRate=1.0
+bfSuper=0.7
+bfExpo=0.0
+
+// KISS
+kissRcRate=0.7
+kissRate=0.7
+kissCurve=0.4
+
+
+// KISS rate function
+var kscalc = function(rcCommand, rate, rcCurve, rcRate) {
+    kissRpyUseRates = 1 - Math.abs(rcCommand) * rate
+    kissRxRaw = rcCommand * 1000
+    kissTempCurve = (kissRxRaw * kissRxRaw / 1000000)
+    rcCommand = ((rcCommand * kissTempCurve) * rcCurve + rcCommand * (1 - rcCurve)) * (rcRate / 10)
+    kissAngle = ((2000.0 * (1.0 / kissRpyUseRates)) * rcCommand) // setpoint is calculated directly here
+    return kissAngle
+}
+
+
+// RF rate function
+var rfcalc = function(rcCommand, rate, expo, acrop) {
+    returnValue = ((1 + 0.01 * expo * (rcCommand * rcCommand - 1.0)) * rcCommand)
+    returnValue = (returnValue * (rate + (Math.abs(returnValue) * rate * acrop * 0.01)))
+    return returnValue
+}
+
+
+// BF rate calculation function
+var bfcalc = function(rcCommand, rcRate, expo, superRate) {
+    var clamp = function(n, minn, maxn) {
+        return Math.max(Math.min(maxn, n), minn);
+    }
+
+    if(rcRate > 2.0)
+        rcRate = rcRate + (14.55 * (rcRate - 2.0))
```

Comment:
This should actually be changed to 14.54 (just double checked in BF code), changing in python as well

Reply:
Okay. I'll change it.